deploy/helm/charts/README.md

```diff
@@ -100,23 +112,22 @@ The following table lists the configurable parameters of the OpenEBS LocalPV Pro
 | `localpv.healthCheck.periodSeconds`         | How often to perform the liveness probe           | `60`                            |
 | `localpv.replicas`                          | No. of LocalPV Provisioner replica                | `1`                             |
 | `localpv.enableLeaderElection`              | Enable leader election                            | `true`                          |
-| `localpv.basePath`                          | BasePath for hostPath volumes on Nodes            | `"/var/openebs/local"`          |
+| `localpv.basePath`                          | Default value of BasePath for hostPath volumes    | `"/var/openebs/local"`          |
 | `localpv.affinity`                          | LocalPV Provisioner pod affinity                  | `{}`                            |
 | `helperPod.image.registry`                  | Registry for helper image                         | `""`                            |
 | `helperPod.image.repository`                | Image for helper pod                              | `"openebs/linux-utils"`         |
 | `helperPod.image.pullPolicy`                | Pull policy for helper pod                        | `"IfNotPresent"`                |
 | `helperPod.image.tag`                       | Image tag for helper image                        | `2.8.0`                         |
+| `hostpathClass.basePath`                    | BasePath for openebs-hostpath storageClass        | `"/var/openebs/local"`          |
```

Comment:
Why do we need this? can we use the localpv.basePath itself in the storage class - avoid users from entering the same details twice?

Reply:
@kmova -- Which of these will be more appropriate here?
1. Make the default storageClass honor the `localpv.basePath` flag, and remove the `hostpathClass.basePath` flag from the docs and values.yaml file. 
2. Trigger the creation of a 'custom-hostpath' storageClass if `localpv.basePath` is used. This will leave the container env and default openebs-hostpath with default values. At the end of the setup there will be 3 storageClasses created instead of 2 -- openebs-hostpath, openebs-device, custom-hostpath. Here also, we will remove the `hostpathClass.basePath` flag.